src/api/routes/controllers/users.controller.js

```diff
@@ -14,7 +15,9 @@ exports.create = async (req, res, next) => {
 
 exports.getGoals = async (req, res, next) => {
   try {
-    const { createdGoals } = req.app.locals.authResult;
+    const userId = req.app.locals.userId;
+    const user = await User.findById(userId);
+    const { createdGoals } = user;
```

Comment:
호찬님께 드렸던 피드백과 비슷한 맥락인데, user의 정보 전체가 필요한 게 아니니 필요한 필드만 가져오면 어떨까요?

```suggestion
    const createdGoals  = await User.findById(userId, "createdGoals");
```

Reply:
불필요하게 모든 값을 가져올 필요없이 필요한 필드만 가져오는게 더 좋겠네요 해당 부분 수정하도록 하겠습니다!

Reply:
https://stackoverflow.com/questions/24348437/mongoose-select-a-specific-field-with-find

제 pr에도 썼는데 반환 값이 객체이고 그 안에, id값이 default로 반환되고, 찾는 필드가 들어갑니다.

구조분해 할당을 쓰셔야 할꺼에요! id값을 빼는 로직은 `"createdGoals" -_id`  입니다.